articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+
+![Example of WLAN](/engineering-education/network-types-topologies/wlan.jpg)
+
+#### Metropolitan Area Network (MAN)
+The point of having [MANs](https://en.wikipedia.org/wiki/Metropolitan_area_network) is similar to LANs. One of the differences is the geographical spread of the network range. As mentioned above, LANs are confined to small areas. But, as the name suggests, MAN’s are spread across an entire metropolitan city. They can be used to share data from one point of the city to another. Business establishments, telephone companies, cable TV’s use this type of networks.  
+
+![Example of MAN](/engineering-education/network-types-topologies/man.jpg)
+
+#### Wide Area Network (WAN)
+This type of network is currently the one mostly used. [WAN](https://en.wikipedia.org/wiki/Wide_area_network) is nothing but a collection of several LANs linked together. These networks are spread across countries and other parts of the world. Some of them are connected through optic fibers while some are connected through satellites. WANs are used by government agencies, multinational companies for exchanging information overseas. Since the network is spread across the world, it is maintained by many firms and companies. When the network is this big, it is prone to failures and system shutdowns once in a while. The biggest WAN in existence is the Internet.   
+
+![Example of WAN](/engineering-education/network-types-topologies/wan.jpg)
+
+ Configuring a computer network can also depend on the extent of the network, the distance between devices and the number of devices used. While LANs, WLANs, and WANs are most commonly used, there are some [other](https://www.belden.com/blog/smart-building/network-types) computer networks that can also be used based on our networking parameters.
+
+
+
+### Network Topologies
+Arrangement of components or parts in a specific way is called a **Topology.** In [network topologies](https://en.wikipedia.org/wiki/Network_topology), the components are arranged in a systematic order for *smooth data flow* in a communication network. Here, the components may be devices, workstations, etc which are also referred to as **nodes**. Many topologies are used based on their applications, structure and efficiency. While some are used by commercial establishments and industrial sectors, some are used by schools and for household purposes.

```

Comment:
Can we unpack this sentence:
"Many topologies are used based on their applications, structure and efficiency." 
I think there is some useful info here - but as a beginner - Im not entirely sure what this sentence means.
Perhaps restructuring the sentence would help. 
If it makes sense we can also omit the sentence- if the meaning and or message is handled somewhere else.

Reply:
Yes- the types of topologies based on their structure, applications are explained in their respective parts. I feel that the sentence is redundant in the intro part, so I'll consider removing it.